tests/dummy/app/app.js

```diff
@@ -3,6 +3,9 @@ import Resolver from './resolver';
 import loadInitializers from 'ember-load-initializers';
 import config from './config/environment';
 
+// ember-browserify requires every package to be imported somewhere in the `app` tree
+import 'npm:common-tags';
```

Comment:
This is still needed because the `new-review-test` route unit tests imports `common-tags`, but ember-browserify doesn't crawl test files when determining what modules to bundle up.